test/build/50_ch-image.bats

```diff
@@ -63,6 +63,27 @@ EOF
     [[ $output = *"00_tiny"* ]]
 }
 
+@test 'ch-image reset' {
+   export CH_IMAGE_STORAGE="$BATS_TMPDIR"/reset
+   run ch-image pull alpine
```

Comment:
Omit `run` unless you're examining the output or checking status for something other than success.